### PR TITLE
registry: Add support for ListRepositoriesV2 and ListManifests commands.

### PR DESCRIPTION
**Pull request by CollinShoop**
CON-3907

Required bump to godo 1.73

### CODE REVIEW

File: commands/registry.go [hunk 1]
```diff
@@ -144,6 +144,21 @@ func Repository() *Command {
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
 	)
 
+	listRepositoriesV2Desc := `This command retrieves information about repositories in a registry, including:
+  - The repository name
+  - The latest manifest of the repository
+  - The latest manifest's latest tag, if any
+  - The number of tags in the repository
+  - The number of manifests in the repository
+`
```
Comment (andrewsomething) [inline commands/registry.go:153]: I wonder if there is some additional info we could provide to users that would help them choose between `list` and `list-v2`?

File: commands/registry.go [hunk 2]
```diff
@@ -144,6 +144,21 @@ func Repository() *Command {
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
```
Comment (andrewsomething) [inline commands/registry.go:144]:
```suggestion
		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
		hiddenCmd(),
```

FYI: If we want to push users towards the v2 command, it is possible to hid the current one from menus while leaving it available in order to not break any scripting.

**Review by andrewsomething — APPROVED** [review]
:+1:  This looks great. 

I did have a question inline about helping users understand the difference between `list` and `list-v2`

File: commands/registry.go (hunk 1 above)
Comment (CollinShoop) [reply commands/registry.go:154]: I'm not sure. I was thinking just the fact of `v2` and seeing the different results would be self explanatory. I'm open to suggestions

Comment (andrewsomething) [reply commands/registry.go:154]: Since we've gone with hiding v1, it probably not very important now.

**Review by andrewsomething — APPROVED** [review]

**Pull request merged**